<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1342</width>
    <height>776</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/searchVehicle.png</normaloff>images/searchVehicle.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Truy vấn:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbb">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Tất cả xe</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Loại vé:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbbCard">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Tất cả loại vé</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vé tháng</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vé lượt</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Loại xe:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbbVehicle">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Tất cả loại xe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Xe máy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ô tô</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Chọn ngày:</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateStart">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy HH:mm</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateEnd">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy HH:mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>đến:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Mã thẻ:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnCard">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Biển số:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnPlate">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>841</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Mã thẻ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Biển số</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Loại vé</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Loại xe</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Chủ xe</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>471</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="lblImg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>451</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid gray;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblCar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
border: 2px solid gray;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Ô tô</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblMtb">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
border: 2px solid gray;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Xe máy</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>700</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1342</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
